src/telemetry/reporter.ts

```diff
@@ -2,7 +2,7 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
-
+/* tslint:disable */
```

Comment:
This is disabling linting for the entire file.  Can we disable individual rules on individual lines via `//tslint:disable-next-line:<rule>`?  Its a little more time-consuming, but it let's us enforce "no new linter errors".

Reply:
This change should be reverted. It's a mistake.